packages/components/src/Gallery/Gallery.test.tsx

```diff
@@ -167,3 +172,33 @@ describe("when a non-image is clicked", () => {
     });
   });
 });
+
+describe("Thumbnails", () => {
+  it.each(files.map(file => [file.name, file.thumbnailSrc]))(
+    "should display the thumbnailSrc as thumbnails for %s",
+    async (fileName, src) => {
+      const { getByAltText } = render(<Gallery files={files} />);
+      const thumbnailImage = getByAltText(fileName);
+
+      await waitFor(() => {
+        expect(thumbnailImage).toHaveAttribute("src", src);
+      });
+    },
+  );
+
+  it.each(files.map(file => [file.name, file.src]))(
+    "should display the src as thumbnails for %s when thumbnailSrc is not provided",
```

Comment:
and then for this one: "should use the src as image source for %s when thumbnailSrc is not provided",